fe/fe-core/src/main/java/org/apache/doris/analysis/IndexDef.java

```diff
@@ -37,6 +37,11 @@ public class IndexDef {
     private String comment;
     private Map<String, String> properties;
 
+    public static final String NGRAM_SIZE_KEY = "gram_size";
+    public static final String NGRAM_BF_SIZE_KEY = "bf_size";
```

Comment:
I think it's by design. The idea mainly from clickhouse, which I think design for throughput. We could switch to set FPR instead of the bf size, but which need keep the distinct values during writer, that's cost too much.
BTW, if use the FPR, the end user may not clear about the cardinality of the values, how about lead to big bloom filter size? which may hurt the performance rather than improve.
So overall, I think it's better to keep the design.